tests/test_jump.py

```diff
@@ -40,23 +45,25 @@ def test_find_simple_circle():
     plane[2, 3] = DQFLAGS['JUMP_DET']
     plane[2, 1] = DQFLAGS['JUMP_DET']
     circle = find_circles(plane, DQFLAGS['JUMP_DET'], 1)
+    assert circle[0][0] == pytest.approx((2, 2))
     assert circle[0][1] == pytest.approx(1.0, 1e-3)
```

Comment:
here is where the fit circle from the `scikit-image` contour differs from the `opencv-python` circle; the radius is 1.5 instead of 1